Workbooks/AzurePurview.json

```diff
@@ -0,0 +1,1452 @@
+            },
+            {
+              "columnMatch": "SourceName",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "SourceType",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "SourcePath",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "SourceSubscriptionId",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "SourceRegion",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "SourceCollectionName",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "SourceOwner",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "AssetName",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "AssetPath",
+              "formatter": 7,
+              "formatOptions": {
+                "linkTarget": "GenericDetails",
+                "linkIsContextBlade": true,
+                "customColumnWidthSetting": "70ch"
+              }
+            },
+            {
+              "columnMatch": "AssetType",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "AssetCreationTime",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "AssetModifiedTime",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "AssetOwner",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "FileExtension",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "FileSize",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "ActivityType",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "ActivityTrigger",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "Classification",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "ClassificationCount",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "SensitivityLabelGuid",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "SensitivityLabel",
+              "formatter": 5
+            },
+            {
+              "columnMatch": "UserId",
+              "formatter": 5
+            }
+          ],
+          "filter": true
+        },
+        "sortBy": []
+      },
+      "conditionalVisibility": {
+        "parameterName": "Tab",
+        "comparison": "isEqualTo",
+        "value": "Labels"
+      },
+      "customWidth": "50",
+      "name": "query - 13",
+      "styleSettings": {
+        "showBorder": true
+      }
+    }
+  ],
+  "fallbackResourceIds": [
```

Comment:
Please remove the reference to Azure resource by making the fallbackResourceIds attribute empty as below.
"fallbackResourceIds": [],

Thanks,
Ruchita